src/main/resources/application.yaml

```diff
@@ -70,6 +70,9 @@ service:
       target-storage-account: bulkscan
       target-container: sscs
 
+reports:
+  recipients: ${REPORTS_RECIPIENTS:}
```

Comment:
is `:` needed?

Reply:
We should only use default values where reasonable default values exist. I guess empty recipients field is always invalid in an email.

Reply:
Will do

Reply:
Ok, it might make sense to have an empty recipient list by default. It will only make sense, though, when host is not configured (as we make the job's presence dependent on it).